app/models/street/StreetEdgeTable.scala

```diff
@@ -174,6 +185,19 @@ object StreetEdgeTable {
     }
   }
 
+  /**
+    * Finds the OSM ID of the street with the street edge ID streetEdgeId.
+    * 
+    * @return the OSM ID of the street with the street edge ID streetEdgeId.
+    */
+  def osmStreetIds(streetEdgeId: Int): Int = db.withSession { implicit session =>
+    val getOsmQuery = Q.queryNA[Int](
```

Comment:
Above, I added a comment about converting this function and query to take in a list of IDs instead of just one. In addition, I'd like you to convert the query over to using [Slick](https://scala-slick.org/doc/2.1.0/queries.html) syntax instead of plain SQL! For the function as-written, it would probably look something like:
```
osmStreetTable.filter(_.streetEdgeId === streetEdgeId).map(_.osmWayId).list.head
```

This would change slightly for taking in and outputting multiple lists of IDs (and you'd probably want to output the full `OsmStreet` object instead of just the IDs when doing a list). There should be examples of this syntax throughout our code, but if you're having trouble finding some or getting it to work, just lmk!